__tests__/app.test.js

```diff
@@ -133,6 +133,26 @@ describe("/api/articles/:article_id", () => {
   });
 })
 
+//ticket 21
+describe('/api/users', () =>{
+  describe('GET', () => {
+    test.only('200 - returns array of objects with username property', () => {
+      return request(app).get('/api/users')
+      .then((res)=>{
+        console.log(res.body, 'res.body')
```

Comment:
You're missing the status expect in this test.

Also - Console.log!

Reply:
thanks, fixed